src/Common/Core/Impl/Telemetry/TelemetryServiceBase.cs

```diff
@@ -32,13 +32,13 @@ public abstract class TelemetryServiceBase : ITelemetryService, IDisposable {
         /// </summary>
         public bool IsEnabled {
             get {
-                return this.TelemetryRecorder.IsEnabled;
+                return this.TelemetryRecorder?.IsEnabled == true;
```

Comment:
Not anymore, sorry. But undoing the change and doing a few full tests runs should flush them out.

---
In reply to: [74832919](https://github.com/Microsoft/RTVS/pull/2173#discussion_r74832919) [](ancestors = 74832919,74832126,74828037)